op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -80,12 +80,18 @@ var (
 			// CreateX
 			common.HexToAddress("0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed"): false,
 		},
+		BaklavaNetworkID: {
+			// DeterministicDeploymentProxy
+			common.HexToAddress("0x4e59b44847b379578588920cA78FbF26c0B4956C"): false,
+		},
```

Comment:
Why do we need only a single entry compared to the many for alfajores?

Reply:
```
Processing address: 0xcA11bde05977b3631167028862bE2a173976CA11
Code mismatch for address: 0xcA11bde05977b3631167028862bE2a173976CA11
Codesize on https://baklava-forno.celo-testnet.org: 0
Nonce on https://baklava-forno.celo-testnet.org: 0
```

From the output we see that `0xcA11bde05977b3631167028862bE2a173976CA11` doesn't have any code or nonce on Baklava, so that account will be overwritten by default.

The same is true for all accounts besides the one listed.